daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -75,6 +75,114 @@ Code | Description
 
 This API does not return any content.
 
+## Raise Event request
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceID>/raiseEvent/<eventName>
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`workflowComponentName` | Current default is `dapr` for Dapr Workflows
+`instanceId` | Unique value created for each run of a specific workflow
+`eventName` | The name of the event to raise
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Request formatted correctly, error in dapr code or underlying component
+
+### Response content
+
+None.
+
+## Pause workflow request
+
+Pause a running workflow instance with the given name and instance ID.
```

Comment:
Nit: Here and below you are saying "with the given name". Since these methods don't require a workflowname, what is this referring to?

Reply:
most likely it's just me copy/pasting and not double checking :) thanks for the catch!